### ISSUE TITLE
min/max on iterables

### ISSUE THREAD

**Issue by stuartarchibald** [numba/numba#3776]
## Feature request

`min` and `max` builtins should work on iterables, e.g.

```
In [23]: @njit 
    ...: def foo(): 
    ...:     return min(range(5)) 
```

**Comment by belm0** [comment]
if possible the fix should also address min/max with 3 or more positional args.  cc: @rjenc29 

at the very least, pysupported.html should note the limitations

**Comment by stuartarchibald** [comment]
@belm0 if 3+ arg support is important to your use case please do open a new issue.

Closing this issue as the OP was fixed by #3820.
